### PR TITLE
add bug report template

### LINKED ISSUE #4: Add a bug report template

### PR DESCRIPTION
**Pull request by dejmedus**
Added bug_report.yml to .github/ISSUE_TEMPLATE

should close #4

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      value: '## Thanks for taking the time to fill out this bug report!'
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: dropdown
```
Comment (mishushakov) [inline .github/ISSUE_TEMPLATE/bug_report.yml:17]: Change this to input

**Review by mishushakov — CHANGES_REQUESTED** [review]
Looks fantastic. The only suggestion I have is to change the version number from dropdown to input, so we don't have to maintain the list

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (dejmedus) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Changed it

**Review by mishushakov — APPROVED** [review]

**Comment by mishushakov** [comment]
Good enough, thanks!

**Comment by mishushakov** [comment]
@dejmedus I also have these two issues that could use a little help:

- #7
- #8

You can subscribe to these issues and I'll continue there

**Comment by sebastianwis** [comment]
Thank you very much, @dejmedus!
That helped a lot! 
Also, we are very happy to have you as our very first contributor to Step CI. 
It would be great if you would like to accompany us on the further journey in any way!

**Pull request merged**